lib/matplotlib/dates.py

```diff
@@ -201,6 +220,65 @@ def _get_rc_timezone():
     MO, TU, WE, TH, FR, SA, SU)
 WEEKDAYS = (MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY)
 
+# default epoch: passed to np.datetime64...
+_epoch = None
+_epoch_used = False
+
+
+def _reset_epoch():
+    """
+    Reset the Matplotlib date epoch to its rcParams value.
+    """
+    global _epoch
+    global _epoch_used
+
+    _epoch_used = False
+    _epoch = matplotlib.rcParams['date.epoch']
+
+
+def set_epoch(epoch):
+    """
+    Set the epoch (origin for dates) for datetime calculations.  The
```

Comment:
```suggestion
    Set the epoch (origin for dates) for datetime calculations.
```
First sentence is a separate paragraph.